geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Bis 14 Tage vor einer Vollversammlung können die einladenden Personen gestellte Anträge in die Tagesordnung aufnehmen.
+
+(4) Bis 7 Tage vor einer Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den einladenden Personen geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
```

Comment:
ÄA: Streichung von (2)

zu 2: Versammlungsleitung kann das über einen GO-Antrag machen. Falls die Mehrheit der Fachschaft Diskussionsbedarf in einem Thema sieht sollte eine Versammlungsleitung sich nicht über die Fachschaft hinwegsetzen können. Ein ausleiern von Diskussionen wird dadurch vermieden, dass die Fachschaft irgenwann keinen Bock mehr auf Diskussionen hat, und Schließung der Rednerliste etc. annimmt.

Reply:
Sorry, verstehe ich nicht. :)

Es geht glaube ich darum, dass die Sitzungsleitung grundsätzlich eher neutral sein soll. Wenn die jetzt aber *unbedingt* zu einem Thema sich äußern oder mitdiskutieren möchte (einfache Äußerung als kurze Anmerkung sollte ja i.O. sein), dann gibt sie halt zur Sicherung der Neutralität ab. Gleichfalls, wenn ein Antrag von einer bestimmten Gruppe stammt, dann kann die ja auch den Teil der Sitzung leiten, wenn förderlich. IMHO.

Reply:
Stimmt, das sollte sich auf (3) beziehen. Wobei eine Gruppe die einen Antrag einbringt ja bei diesem Antrag tendenziell genau nicht Neutral sind, wenn es nicht grad um ein Meinungsbild geht. Sie haben den Antrag ja eingebracht.

Reply:
Ah okay. Das ergibt mehr Sinn. Grundsätzlich kenne ich es von Sitzungen so, dass die Antragstellenden den Antrag vorstellen und dann die Diskussion anleiten, aber das hängt halt vom Thema und der Debattenkultur ab, die ich (noch) nicht gut einschätzen kann. An sich finde ich es gut, dass die Sitzungsleitung eine Redeliste von sich aus führen kann, Redezeitbegrenzung kann man problematischer sehen. Eine Versammlung lebt ganz viel auch von "inoffiziellen Sachen", wo jetzt nicht auf jede Formalie gepocht wird. Dabei fehlt als Gegengewicht natürlich die Befugnis einer qualifizierten Minderheit (also in diesem Fall: 1/3 der VV), die Redezeitbegrenzung/-liste aufzuheben. Wobei ich bei einer Redeliste das ganze nicht als pauschales Vetorecht, sondern eher als Teilhaberecht ausgestalten würde:

> Auf Antrag von 10 Teilnehmenden (oder 10%, siehe andere Nachricht) kann, auf Antrag von einem Drittel der Anwesenden muss die Vollversammlung eine bestimmte Person auf eine bestimmte Position auf der Redeliste setzen. Unzulässig ist der Antrag, wenn dadurch das Verhältnis an Für- und Widersprechenden unverhältnismäßig verzerrt werden würde. Das Recht steht der qualifizierten Minderheit auch dann nicht zu, wenn diese Geschäftsordnung oder die Satzung andere Regelungen für die Verteilung der Wortmeldungen trifft.

Reply:
Ziel der Abgabe der Versammlungsleitung ist es, in angemessenen Punkten, z.B. dem Bericht aus den studentischen Gremien, an andere zu übergeben, die sich auf diese Punkte vorbereitet haben und die Diskussion kompetent führen können. Zum Job der Versammlungsleitung gehört mMn ebenso, einzuschätzen, wann das sinnvoll ist. Bei einem Antrag die Sitzungsleitung an die den Antrag stellende Person abzugeben, ist im Zweifel nicht sinnvoll. Wenn die Mehrheit glaubt, dass die Versammlungsleitung hier nicht sinnvoll agiert, kann das a) zum Ausdruck gebracht werden und führt im Zweifel b) dazu, dass sich eine entsprechende Mehrheit findet, die wie oben beschrieben Einfluss auf den Verlauf der Sitzung haben kann

Die Führung einer Redeliste würde ich nicht immer über einen GO-Antrag machen, das wäre ziemlich umständlich für die Versammlungsleitung.
Den GO-Antrag für Redezeitbegrenzung würde ich mitnehmen in die FSR-Runde. Ich sehe die Vorteile, als Nachteil sehe ich aber auch, dass solche Anträge gerne zum Ausdruck von Agressionen genutzt werden, die nicht hilfreich für eine konstuktive Diskussion sind.

Reply:
Wir haben auf der letzten Sitzung entschieden, einen GO-Antrag auf Festsetzung einer Redezeitbregenzung einzuführen und der Sitzungsleitung das explizite Recht zu nehmen, siehe letzten Commit. Kann dieser Thread damit resolved werden?